### PR TITLE
Add HEMTT to build.py

### PR DESCRIPTION
**Pull request by neilzar**
**When merged this pull request will:**
- Use HEMTT when running build.py, if it is installed.

### CODE REVIEW

File: tools/build.py [hunk 1]
```diff
@@ -9,6 +9,17 @@
 PREFIX = "zen_"
 ##########################
 
+def tryHemttBuild(projectpath):
+    hemttExe = os.path.join(projectpath, "hemtt.exe")
```
Comment (CreepPork) [inline tools/build.py:13]: What if I would want to build it on Linux too?

File: tools/build.py [hunk 2]
```diff
@@ -9,6 +9,17 @@
...
+    if os.path.isfile(hemttExe):
+        os.chdir(projectpath)
+        ret = subprocess.call([hemttExe, "pack"], stderr=subprocess.STDOUT)
+        print("Using hemtt: {}".format(ret));
+        return True
+    else:
+        print("hemtt not installed");
```
Comment (CreepPork) [inline tools/build.py:20]:
```suggestion
        print("HEMTT is not available");
```

File: tools/build.py [hunk 3]
```diff
@@ -9,6 +9,17 @@
 PREFIX = "zen_"
 ##########################
 
+def tryHemttBuild(projectpath):
+    hemttExe = os.path.join(projectpath, "hemtt.exe")
+    if os.path.isfile(hemttExe):
+        os.chdir(projectpath)
+        ret = subprocess.call([hemttExe, "pack"], stderr=subprocess.STDOUT)
+        print("Using hemtt: {}".format(ret));
```
Comment (CreepPork) [inline tools/build.py:17]:
```suggestion
        print("Using HEMTT: {}".format(ret));
```

File: tools/build.py (hunk 1 above)
Comment (neilzar) [reply tools/build.py:13]: I took this code from the CBA repo, I don't have enough experience with Linux and its file system, or with python, to know how to make it work.

Comment (jonpas) [reply tools/build.py:13]: You shouldn't use `build.py`, but use `hemtt` binary directly.

Comment (CreepPork) [reply tools/build.py:13]: Yep, that's true. I'd guess I'm just overthinking it too much.

**Review by CreepPork — APPROVED** [review]

**Review by Kexanone — APPROVED** [review]

**Review by mharis001 — APPROVED** [review]

**Pull request merged**